sstables/mx/bsearch_clustered_cursor.hh

```diff
@@ -526,6 +526,22 @@ public:
         });
     }
 
+    future<position_range> full_clustering_range() override {
+        if (_blocks_count == 0) {
+            co_return position_range::all_clustered_rows();
+        }
+
+        auto first_block = co_await _promoted_index.get_block(0, _trace_state);
+        auto last_block = co_await _promoted_index.get_block(_blocks_count - 1, _trace_state);
+
+        auto first_ckey_prefix = first_block->start->key(); // very first ckey prefix for current partition
+        auto last_ckey_prefix = last_block->end->key(); // very last ckey prefix for current partition
```

Comment:
Please verify that this includes range tombstones.

Reply:
Alright.